cpp/dolfinx/fem/FiniteElement.h

```diff
@@ -170,7 +174,8 @@ class FiniteElement
   /// nodal positions. For other elements the points will typically be
   /// the quadrature points used to evaluate moment degrees of freedom.
   /// @return Points on the reference cell. Shape is (num_points, tdim).
```

Comment:
I guess `@return` docstring should be slightly modified.
Same remark regarding vector vs span in returntype